sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -395,7 +395,7 @@ export class MessageSender extends LinkEntity {
       operation: sendEventPromise,
       connectionId: this._context.namespace.connectionId!,
       operationType: RetryOperationType.sendMessage,
-      times: Constants.defaultRetryAttempts,
+      maxRetries: Constants.defaultMaxRetriesForConnection,
```

Comment:
This is a wrong replacement. The default for connection retries is being used instead of the default for operation retries

Reply:
Oops, copy paste error - thanks!